src/mux-analytics.brs

```diff
@@ -1042,13 +1088,19 @@ function muxAnalytics() as Object
     else if eventType = "Complete"
       ' Complete signals an ad has finished playback
       m._Flag_rssAdEnded = true
+      if m._lastAdResumeTime <> Invalid
+        m._adWatchTime += max(0, now - m._lastAdResumeTime)
+        m._lastAdResumeTime = Invalid
+      end if
+      m._totalAdWatchTime += m._adWatchTime
       m._addEventToQueue(m._createEvent("adended"))
     else if eventType = "Impression"
       ' When an additional ad is played within an ad pod, we do not get
       ' the AdStateChange events or anything other than the Impression
       ' event to know that a new ad was played
       if m._Flag_rssAdEnded
         m._Flag_rssAdEnded = false
+        m._lastAdResumeTime = now
```

Comment:
I _think_ we need to reset `m._adWatchTime` here also. This is the only indication that we get for a new ad within a pod, so we want to reset the counter here or we'll end up counting the playing time for the first ad for every subsequent ad and on and on.